exopy/tasks/configs/base_configs.py

```diff
@@ -47,15 +50,15 @@ class BaseTaskConfig(Atom):
 
     def __init__(self, **kwargs):
         super(BaseTaskConfig, self).__init__(**kwargs)
-        # Force check to ensure that the possible default value of task_name
-        # is tested.
-        self.check_parameters()
```

Comment:
From my point of view this could be problematic in subclasses if we need to validate something else than the name. It is not the job of the default value builder to validate the name.

Reply:
I needed to remove the check_parameters to prevent an infinite loop. I can't exactly remember why it happened but moving the check from __init__ to _default_task_name solved the issue.
I can look at this more closely if you want.

Reply:
It seems that reworking the PR fixed the issue so I am putting the \check_parameters` back.